bin/doctrine.php

```diff
@@ -0,0 +1 @@
+../vendor/doctrine/orm/bin/doctrine.php
```

Comment:
This file shouldn't be in here, it should be ignored - a .gitignore will need to be added to the repo.